src/main/java/com/google/api/gax/core/FlowControlSettings.java

```diff
@@ -49,12 +49,23 @@ public static FlowControlSettings getDefaultInstance() {
   @Nullable
   public abstract Integer getMaxOutstandingRequestBytes();
 
+  /** Is flow control enabled. Defaults to true. */
+  public abstract boolean getIsEnabled();
```

Comment:
While I understand the desire for this flag, I think this complicates the relationship between this flag and the `maxOutstanding*` methods. Does it override those options? If so, it should be documented.